lib/registries/addon/components/registries-states/component.ts

```diff
@@ -0,0 +1,22 @@
+import { computed } from '@ember-decorators/object';
+import { service } from '@ember-decorators/service';
+import Component from '@ember/component';
+import I18N from 'ember-i18n/services/i18n';
+import defaultTo from 'ember-osf-web/utils/default-to';
+import styles from './styles';
+import layout from './template';
+
+export default class RegistratiesStates extends Component {
+    styles = styles;
+    layout = layout;
```

Comment:
[There's a decorator for this!](https://github.com/CenterForOpenScience/ember-osf-web/blob/develop/app/decorators/component.ts#L36)